ContentProtection/BasicPlayReady/pom.xml

```diff
@@ -14,9 +14,9 @@
       <version>1.1.1</version>
     </dependency>
     <dependency>
-      <groupId>com.microsoft.azure.mediaservices.v2018_07_01</groupId>
+      <groupId>com.microsoft.azure.mediaservices.v2020_05_01</groupId>
       <artifactId>azure-mgmt-media</artifactId>
-      <version>1.0.0-beta-4</version>
+      <version>1.0.0-beta</version>
```

Comment:
>- [](start = 20, length = 1)

Why is this still beta? How is this version used?

Reply:
will follow up. but that will not affect this PR, so merging.